### PR TITLE
add license file

### PR DESCRIPTION
**Pull request by wangxinyi7**
DoD

- [x] Add license file into ZIP
- [x] Add license file into linux config_dir
- [x] Add license file into docker image
- [x] Add license label into docker image

### CODE REVIEW

**Review by blake — APPROVED** [review]

**Comment by wangxinyi7** [comment]
@blake Thanks for approving the PR. Since I don't have the merging power, if you happen to be able to merge this PR, could you please merge it? Thanks!

**Comment by wangxinyi7** [comment]
Hi Lorna @lornasong, sorry to bump you here, I know you are no longer in consul but I see you are the top committer in this repo, since I don't have merge power, could you please merge this PR if you still have the merge permission? Thanks!

**Comment by lornasong** [comment]
Hey @wangxinyi7 I unfortunately don't have merge permissions anymore myself:

image

**Pull request merged**
